src/test/java/uk/gov/hmcts/reform/sendletter/services/LetterServiceTest.java

```diff
@@ -1,59 +1,86 @@
 package uk.gov.hmcts.reform.sendletter.services;
 
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.microsoft.azure.servicebus.primitives.ServiceBusException;
 import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.mockito.Mock;
 import org.mockito.runners.MockitoJUnitRunner;
 import uk.gov.hmcts.reform.sendletter.SampleData;
 import uk.gov.hmcts.reform.sendletter.model.Letter;
-import uk.gov.hmcts.reform.sendletter.notify.INotifyClient;
+import uk.gov.hmcts.reform.sendletter.queue.producer.QueueProducerClient;
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.catchThrowable;
 import static org.mockito.Matchers.any;
-import static org.mockito.Mockito.doThrow;
-import static org.mockito.Mockito.times;
-import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
 
 @RunWith(MockitoJUnitRunner.class)
 public class LetterServiceTest {
 
-    @Mock private INotifyClient notifyClient;
+    @Mock private QueueProducerClient queueProducerClient;
 
     private final Letter letter = SampleData.letter();
 
     private LetterService service;
 
     @Before
     public void setUp() {
-        service = new LetterService(notifyClient);
+        service = new LetterService(queueProducerClient);
     }
 
     @Test
-    public void should_send_a_request_to_notify_if_letter_was_not_sent_previously() {
+    public void shouldThrowServiceBusExceptionWhenQueueClientFailsToConnect() throws Exception {
```

Comment:
let's keep the underscores for readability

Reply:
to be honest both are fine to me but I will update it to use underscore pattern.If you look at spring boot test cases they use camel case and if you refer mockito library they use underscore.